server/test/jobs/DurableJobRegistryTest.java

```diff
@@ -0,0 +1,56 @@
+package jobs;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertThrows;
+
+import models.PersistedDurableJob;
+import org.junit.Test;
+
+public class DurableJobRegistryTest {
+
+  @Test
+  public void registeringAndAccessingAJob() throws Exception {
+    var registry = new DurableJobRegistry();
+
+    registry.register(DurableJobName.OLD_JOB_CLEANUP, new FakeJobFactory());
+
+    assertThat(registry.get(DurableJobName.OLD_JOB_CLEANUP))
+        .isInstanceOf(DurableJobRegistry.RegisteredJob.class);
```

Comment:
Remove this as it is redundant with with signature of .get(), the implicit assert is that it doesn't throw an exception.

Reply:
Done